### PR TITLE
castle-view-image: Add version 2.0.0

### PR DESCRIPTION
**Pull request by gheoan**

### CODE REVIEW

File: bucket/glviewimage.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "description": "glViewImage is an image viewer, converter and even a very limited image editor.",
+  "homepage": "https://castle-engine.io/glviewimage.php",
+  "license": "GPL-2.0-or-later",
+  "version": "1.8.0",
+  "checkver": {
+    "github": "https://github.com/castle-engine/glviewimage"
+  },
+  "url": "https://downloads.sourceforge.net/castle-engine/glviewimage-1.8.0-win-i386.zip",
+  "hash": [
```
Comment (Ash258) [inline bucket/glviewimage.json]: Remove, sha256 and md5 hashes

**Review by Ash258 — DISMISSED** [review]
Add autoupdate

**Comment by issaclin32** [comment]
I updated the manifest by myself because this PR has not been updated in a while.
NOTES:
* The software is renamed as *Castle View Image* by the developer.
* `persist` is not needed because the config file is stored at `$Env:LocalAppdata\castle-view-image`.

**Comment by niheaven** [comment]
Should suggest `imagemagick` for `convert` (https://castle-engine.io/castle-view-image.php#section_depends)

**Review by rashil2000 — APPROVED** [review]

**Comment by issaclin32** [comment]
> Should suggest `imagemagick` for `convert` (https://castle-engine.io/castle-view-image.php#section_depends)

[Done.](https://github.com/ScoopInstaller/Extras/blob/master/bucket/castle-view-image.json)

By the way, *imagemagick*'s `convert.exe` does not exist anymore. It is now `magick.exe convert`. However it seems that *castle view image* has not updated their mechanism.

**Pull request merged**